README.md

```diff
@@ -72,9 +72,36 @@ Importing the Gradle project into Intellij and Eclipse both work.
 
 `./gradlew dependencyUpdates`
 
+
+## Run on OpenShift
```

Comment:
I would prefer to have this section in another tutorial or wiki page, as we don't have direct reference to other providers on the README.
However, we could add a section "How to run in production", linking the tutorials on the website. WDYT?

Reply:
sure, I can move it out of the main README and just link to it.  But I'm personally [not a huge fan of keeping doc in a separate repo](https://github.com/alfio-event/alf.io) .. IMHO it's easier to have doc as an integral part of a PR together with code changes. I'm therefore going to propose a new `docs/` folder here - hope you'll like that.